### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/apigee-registry/commit/1340fd20b3eb50baa1a24504c7c48cde14d47ec7#commitcomment-84945593

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/apigee-registry/commit/8683661af799c431b1f28ae5c783cd520c7046fe

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: cmd/registry-server/main.go [hunk 1]
```diff
@@ -99,7 +99,7 @@ func main() {
 		if err != nil {
 			bootLogger.WithError(err).Fatal("Failed to open config file")
 		}
-		// Expand environment variables before unmarshaling.
+		// Expand environment variables before unmarshalling.
```
Comment (jsoref) [inline cmd/registry-server/main.go:102]: This repository uses both spellings roughly evenly, but there are more instances of the double-l spelling in user facing messages...

File: README.md [hunk 2]
```diff
@@ -139,7 +139,7 @@ This software is licensed under the Apache License, Version 2.0. See
 
 ## Disclaimer
 
-This is not an official Google product. Issues filed on Github are not subject
+This is not an official Google product. Issues filed on GitHub are not subject
```
Comment (jsoref) [inline README.md:142]: brand

File: cmd/registry/cmd/upload/testdata/cloudtasks/v2/openapi.yaml [hunk 3]
```diff
@@ -1143,7 +1143,7 @@ components:
...
...ps processing depends on the worker. For example, if the worker is stuck, it may not react to cancelled requests. The default and maximum values depend on the type of request: * For HTTP tasks, the default is 10 minutes. The deadline must be in the interval [15 seconds, 30 minutes]. * For App Engine tasks, 0 indicates that the request has the default deadline. The default deadline depends on the [scaling type](https://cloud.google.com/appengine/docs/standard/go/how-instances-are-managed#instance_scaling) of the service: 10 minutes for standard apps with automatic scaling, 24 hours for standard apps with manual and basic scaling, and 60 minutes for flex apps. If the request deadline is set, it must be in the interval [15 seconds, 24 hours 15 seconds]. Regardless of the task's `dispatch_deadline`, the app handler will not run for longer than the service's timeout. We recommend setting the `dispatch_deadline` to at most a few seconds more than the app handler's timeout. For more information see [Timeouts](https://cloud.google.com/tasks/docs/creating-appengine-handlers#timeouts). `dispatch_deadline` will be truncated to the nearest millisecond. The deadline is an approximate deadline."
```
Comment (jsoref) [inline cmd/registry/cmd/upload/testdata/cloudtasks/v2/openapi.yaml:1146]: Sometimes doubled words are the right thing, this doesn't appear to be such an instance:
```diff
-longer than than the 
+longer than the 
```

File: cmd/registry/diff/differ.go [hunk 4]
```diff
@@ -87,7 +87,7 @@ func addToDiffProto(diffProto *rpc.Diff, changePath *change) {
 	}
 }
 
-// getChanges creates a protodif report from a diff.Diff struct.
+// getChanges creates a protodiff report from a diff.Diff struct.
```
Comment (jsoref) [inline cmd/registry/diff/differ.go:90]: I looked and couldn't find a `protodif` and guessed it was a typo for `protodiff`, but I could be wrong.

File: demos/disco.sh [hunk 5]
```diff
@@ -60,7 +60,7 @@ registry rpc get-api-spec \
 registry rpc get-api-spec --json \
 	--name projects/$PROJECT/locations/global/apis/translate/versions/v3/specs/discovery.json
 
-# You might notice that that didn't return the actual spec. That's because the spec contents
+# You might notice that didn't return the actual spec. That's because the spec contents
```
Comment (jsoref) [inline demos/disco.sh:63]: doubled that is correct more often than most doubled words, but it often isn't strictly necessary, and this appears to be such a case. Happy to drop if I'm wrong.

File: google/cloud/apigeeregistry/v1/scoring/severity.proto [hunk 6]
```diff
@@ -37,6 +37,6 @@ enum Severity {
   WARNING = 2;
 
   // This level indicates that the value of the score is a major red flag and
-  // ideally users should activaly take actions to get to a WARNING or OK level.
+  // ideally users should actively take actions to get to a WARNING or OK level.
   ALERT = 3;
 }
```
Comment (jsoref) [inline google/cloud/apigeeregistry/v1/scoring/severity.proto:42]: I try very hard not to touch anything beyond what I'm changing, even though I join git in disliking missing newline-at-eof.

File: server/registry/internal/storage/list.go [hunk 7]
```diff
@@ -8,7 +8,7 @@
 //
 // Unless required by applicable law or agreed to in writing, software
 // distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or impliec.
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
```
Comment (jsoref) [inline server/registry/internal/storage/list.go:11]: Lawyers tend to get annoyed at people changing licenses...

File: server/registry/internal/storage/models/version.go [hunk 8]
```diff
@@ -28,7 +28,7 @@ type Version struct {
 	Key         string    `gorm:"primaryKey"`
 	ProjectID   string    // Uniquely identifies a project.
 	ApiID       string    // Uniquely identifies an api within a project.
-	VersionID   string    // Uniquely identifies a version wihtin a api.
+	VersionID   string    // Uniquely identifies a version within a api.
```
Comment (jsoref) [inline server/registry/internal/storage/models/version.go:31]: Technically this should be `an api`, but while it's theoretically possible for my tool to complain about this, I haven't tried encouraging it to do so, which means I'd only make the change if I were more awake than I am.

**Review by jsoref — COMMENTED** [review]
Most corrections were automatically suggested by Google Sheets. All fault mine.

File: server/registry/internal/storage/models/version.go (hunk 8 above)
Comment (timburks) [reply server/registry/internal/storage/models/version.go:31]: Thanks, we'll address that separately.

**Review by timburks — APPROVED** [review]
@jsoref Thanks for generating these fixes!

File: demos/disco.sh (hunk 5 above)
Comment (theganyo) [reply demos/disco.sh:63]: Yup, it's correct both ways, so less is more!

**Review by theganyo — APPROVED** [review]

**Pull request merged**